num2words/lang_EN_IN.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2003, Taro Ogawa.  All Rights Reserved.
+# Copyright (c) 2013, Savoir-faire Linux inc.  All Rights Reserved.
+
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+# MA 02110-1301 USA
+
+from __future__ import unicode_literals
+from .lang_EN import Num2Word_EN
+    
+class Num2Word_EN_IN(Num2Word_EN):
+    def set_high_numwords(self, high):
+        self.cards[10**7] = "crore"
+        self.cards[10**5] = "lakh"
+
+    def setup(self):
```

Comment:
What is the purpose of those 3 lines? If I look at the code, the purpose of setting `self.high_numwords` seems to be so it is passed as a `high` argument in `set_high_numwords()`, but you don't use it in your override and the superclass' method is never called.